coalib/output/ConsoleInteraction.py

```diff
@@ -373,26 +368,24 @@ def print_affected_files(console_printer,
             if (
                     sourcerange.file is not None and
                     sourcerange.file not in file_dict):
-                log_printer.warn('The context for the result ({}) cannot '
-                                 'be printed because it refers to a file '
-                                 "that doesn't seem to exist ({})"
-                                 '.'.format(result, sourcerange.file))
+                warnings.warn('The context for the result ({}) cannot '
```

Comment:
No, actually we're not checking the logs for this specifically (also no tests are there for that) so I have used warnings here (It doesn't log the message). As you see its kind-of demo PR, so I'll modify the tests sometime later. :P